docker-compose.yml

```diff
@@ -1,11 +1,37 @@
-app:
-  mem_limit: 300m
-  memswap_limit: 1g
-  env_file:
-    .env
-  build: .
-  ports:
-    - '3000:8000'
-  volumes:
-    - .:/home/nodejs/app
-    - /home/nodejs/app/node_modules
+version: '2'
+
+services:
+  mongodb:
+    container_name: ya-mongo-db
+    image: alexpunct/mongo:3.4
+    volumes:
+      - mongostorage:/data/db
+    ports:
+      - "${MONGODB_EXPOSED_PORT}:27017"
+    environment:
+      - MONGO_DATA_DIR=/data/db
+      - MONGO_LOG_DIR=/dev/null
+      - MONGODB_ADMIN_USER=${MONGODB_ADMIN_USER}
+      - MONGODB_ADMIN_PASS=${MONGODB_ADMIN_PASS}
+      - MONGODB_APPLICATION_DATABASE=${MONGODB_APPLICATION_DATABASE}
+      - MONGODB_APPLICATION_USER=${MONGODB_APPLICATION_USER}
+      - MONGODB_APPLICATION_PASS=${MONGODB_APPLICATION_PASS}
```

Comment:
I think those variables you can put in `something.env` file. Whereas `MONGODB_EXPOSED_PORT` shoulb be in '.env'. And build the DSN of `ME_CONFIG_DBSTRING` variable inside the application

Reply:
look here: https://github.com/oleg-koval/ya-skeleton/pull/10#discussion_r172437855